src/application.ts

```diff
@@ -47,9 +46,11 @@ export class PocketGatewayApplication extends BootMixin(
     // Requirements; for Production these are stored in GitHub repo secrets
     //
     // For Dev, you need to pass them in via .env file
-    const environment: string = process.env.NODE_ENV || 'production';
+    let environment: string = process.env.NODE_ENV ?? 'production';
```

Comment:
Any reason we prefer `let` to `const`?

Reply:
Oops. Vestigial `let` from when I was trying to diagnose why it wasn't accepting my environment variable as being production.